### PR TITLE
CS Fixes

### PR DESCRIPTION
**Pull request by shadyvb**
@westonruter Remaining fixes are mostly false positives regarding indentation issues and usage of _SERVER vars which probably doesn't need to be sanitized.

Wanna know your opinion on this before i start working on finding solutions to skip or retaliate on WPCS.

### CODE REVIEW

File: dependency-minification.php [hunk 1]
```diff
@@ -291,7 +291,7 @@ static function admin_ajax_handler() {
 
 		$redirect_url = add_query_arg( 'page', self::ADMIN_PAGE_SLUG, admin_url( self::ADMIN_PARENT_PAGE ) );
 		$redirect_url = add_query_arg( 'updated-count', $updated_count, $redirect_url );
-		$redirect_url = add_query_arg( 'updated-action', $_REQUEST['depmin_task'], $redirect_url );
+		$redirect_url = add_query_arg( 'updated-action', esc_attr( $_REQUEST['depmin_task'] ), $redirect_url );
```
Comment (westonruter) [inline dependency-minification.php]: @shadyvb `esc_attr`? That doesn't make sense here, right?

**Comment by westonruter** [comment]
@shadyvb are you checking against WPCS `develop` (your recent work) or `master`?

**Comment by shadyvb** [comment]
@westonruter against develop

File: dependency-minification.php (hunk 1 above)
Comment (shadyvb) [inline dependency-minification.php]: yup, changed to esc_html

**Pull request merged**